cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/classification-task-bankmarketing-pipeline.yml

```diff
@@ -25,21 +25,22 @@ jobs:
     limits:
       max_trials: 1
       max_concurrent_trials: 1
-    target_column_name: y
+    target_column_name: "y"
```

Comment:
why do we have automl changes in this PR

Reply:
not sure, I'll revert this